www/viewgame

```diff
@@ -506,6 +506,110 @@ if ($mediantime > 60) {
 
+//----------------Begin finding download links-------------------
+
+// Check to see if the game is available for download in any form.
+// PC == "parchment capable" - a format that we can play with Parchment.
+// HEX = Hugo file playable online with textadventures.online/play/?story=XXX
+$foundGame = $foundHtml = $foundPC = $foundHex = $htmlUrl = $pcUrl = $foundAdrift = $adriftUrl = $hexUrl = false;
+foreach ($links as $link)
+{
+    // if it's a game, note that we found a game link
+    if ($link['isgame'])
+        $foundGame = true;
+
+    $unboxUrl = false;
+    if ($link['compression'] && $link['compressedprimary']) {
+        $open = urlencode($link['compressedprimary']);
+        if (preg_match('!^https?://(www\.|mirror\.)?ifarchive\.org/!', $link['url'])) {
+            $linkUrl = urlencode($link['url']);
+            $unboxUrl = "https://unbox.ifarchive.org/?url=$linkUrl&open=$open";
+        } else {
+            $springThingUrl = preg_replace('!^https?://(www\.)?springthing\.net/(\d+)/files/(.*)$!',
+                'https://ifarchive.org/if-archive/games/springthing/$2/$3', $link['url']);
+            if ($springThingUrl !== $link['url']) {
+                $springThingUrl = urlencode($springThingUrl);
+                $unboxUrl = "https://unbox.ifarchive.org/?url=$springThingUrl&open=$open";
+            } else {
+                $springThingUrl = preg_replace('!^https?://(www\.)?springthing\.net/(\d+)/stories/[^/]+/(.*)$!',
+                    'https://ifarchive.org/if-archive/games/springthing/$2/$3', $link['url']);
+                if ($springThingUrl !== $link['url']) {
+                    $springThingUrl = urlencode($springThingUrl);
+                    $unboxUrl = "https://unbox.ifarchive.org/?url=$springThingUrl&open=$open";
+                }
+            }
+        }
+    }
+
+    if ($link['fmtexternid'] == 'hypertextgame')
+    {
+        if (!$link['compression']) {
+            $foundHtml = true;
+            $htmlUrl = $link['url'];
+            break;
+        } else if ($unboxUrl) {
+            $foundHtml = true;
+            $htmlUrl = $unboxUrl;
+            break;
+        }
+    }
+
+    if (stripos($link['title'], "Play Online") !== false)
+    {
+        $foundGame = $foundHtml = true;
+        $htmlUrl = $link['url'];
+        break;
+    }
+
+    // if this is the first Parchment-capable link, and it's not compressed,
+    // note it so that we can set up a Parchment play link to it
+    if (in_array($link['fmtexternid'], [
+        'zcode',
+        'blorb/zcode',
+        'glulx',
+        'blorb/glulx',
+        'tads2',
+        'tads3',
+        'hugo']))
+    {
+        if (!$link['compression']) {
+            $foundPC = true;
+            $pcUrl = urlencode(urlToMirror($link['url']));
+            break;
+        } else if ($unboxUrl) {
+            $foundPC = true;
+            $pcUrl = urlencode($unboxUrl);
+            break;
+        }
```

Comment:
`$pcUrl` should be modified to have the `https://iplayif.com/?story=` prefix here, instead of what is now two places.

Reply:
Basically, it should work like the adrift code.

Reply:
Ok, I changed this.